frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -61,12 +73,16 @@ export const OngoingMissionCard = ({ mission }: MissionProps): JSX.Element => {
                     missionStatus={mission.status}
                 />
             </TopContent>
-            <BottomContent>
-                <MissionStatusDisplayWithHeader status={mission.status} />
-                <MissionProgressDisplay mission={mission} />
-                <MissionRobotDisplay mission={mission} />
-                <BatteryStatusDisplay batteryLevel={mission.robot.batteryLevel} />
-            </BottomContent>
+            <StyledContent>
+                <MiddleContent>
+                    <MissionStatusDisplayWithHeader status={mission.status} />
+                    <MissionProgressDisplay mission={mission} />
+                </MiddleContent>
+                <BottomContent>
+                    <MissionRobotDisplay mission={mission} />
```

Comment:
Should we keep the old configuration if the screen is large enough? MiddleContent and BottomContent on the same line?